src/otx/engine/engine.py

```diff
@@ -804,6 +807,14 @@ def _build_trainer(self, **kwargs) -> None:
         """Instantiate the trainer based on the model parameters."""
         if self._cache.requires_update(**kwargs) or self._trainer is None:
             self._cache.update(**kwargs)
+            # set up xpu device
+            if self._device.accelerator == DeviceType.xpu:
+                self._cache.update(strategy="xpu_single")
+                # add plugin for Automatic Mixed Precision on XPU
+                if kwargs["precision"] == 16:
+                    self._cache.update(plugins=[MixedPrecisionXPUPlugin()])
```

Comment:
As you know, loss of yolox_tiny model become NaN after some iteration in OTX 1.X, and mixed precision is disabled in OTX 1.X for that reason. Is it fixed in OTX 2.0?

Reply:
I haven't faced this issue during benchmarking otx2.0. Yolox_tiny used in Geti works OK with this plugin

Reply:
![image](https://github.com/openvinotoolkit/training_extensions/assets/92974184/36cf881e-eee8-470c-a12b-28268179179f)
It seems that yolox_tiny still have a problem. How about disabling mixed precision training w/ XPU?

Reply:
What dataset do you use? Is it from tests/assest? Do you propose to remove it for all models?

Reply:
![image](https://github.com/openvinotoolkit/training_extensions/assets/68103869/7ab50e1a-cb1d-4aed-ab0f-97bb35d743db)

Reply:
YOLOX cat train well on all 3 regression dataset in my case

Reply:
I will check the speed difference between FP16 and FP32 on XPU. If there is no difference - let's remove at all